ambry-router/src/main/java/com/github/ambry/router/PutOperation.java

```diff
@@ -2077,6 +2084,10 @@ void poll(RequestRegistrationCallback<PutOperation> requestRegistrationCallback)
       }
     }
 
+    private boolean isS3MultipartPartUpload() {
```

Comment:
When we have the MultiPart code, we may set a flag called "Skip-Composite-Blob" flag.
So here we rely on this flag to skip the compositeBlob step.